site/app/templates/grading/electronic/Status.twig

```diff
@@ -731,23 +731,68 @@
         {% if graded_percentage != -1 or core.getUser().accessFullGrading() or peer %}
             {% if core.getUser().getGradingRegistrationSections()|length != 0 %}
                 {%  if team_assignment %}
-                    <a class="btn btn-primary"
-                       href="{{ download_zip_url }}">
-                        Download Zip of Assigned Teams
-                    </a>
+                    <details class="btn btn-primary" style="padding-right:0px">
+                        <summary float="left">
+                            <a id="zip_link" style="text-decoration:none;color:white;padding:10px;padding-left:0px" 
+                                href="{{ download_zip_url }}">
+                                Download Zip of Assigned Teams
+                            </a>
+                        </summary>
+
+                        <details-menu>
+                            <div>
+                                <form style="text-align: left;padding-top:8px;">
+                                    <input type="checkbox" id="results_folder" onclick="editZipDownload( '{{ download_zip_url }}', true )">
+                                    <label for="results_folder"> Include Results Folder</label><br>
+                                    <input type="checkbox" id="active_version" onclick="editZipDownload( '{{ download_zip_url }}', true )">
+                                    <label for="active_version"> Only Current Active Version</label><br>
+                                </form>
+                            </div>
+                        </details-menu>
```

Comment:
What is the purpose of the `<details-menu>` elements?  They don't seem to be referenced anywhere else in the codebase.